LocationSimulator/ViewController/MapViewController/MapViewController+LocationSpooferDelegate.swift

```diff
@@ -90,7 +90,6 @@ extension MapViewController: LocationSpooferDelegate {
 
         // Post the update for the current app status.
         NotificationCenter.default.post(name: .StatusChanged, object: self, userInfo: [
-            "device": spoofer.device,
```

Comment:
userInfo["device"] wasn't used in the observer, so to simplify things I just removed this.